examples/ppo/ppo_lib.py

```diff
@@ -274,31 +296,39 @@ def train(
   log_frequency = 40
   checkpoint_frequency = 500
 
-
-  for s in range(loop_steps):
+  initial_params = get_initial_params(jax.random.PRNGKey(0), model)
+  state = create_train_state(initial_params, model, config,
+                             loop_steps * config.num_epochs)
+  del initial_params
+  state = checkpoints.restore_checkpoint(model_dir, state)
+  # number of train iterations done by each train_step
+  iterations = config.num_agents * config.actor_steps // config.batch_size
+  start_step = int(state.step) // config.num_epochs // iterations
+  logging.info(f'Start training from step: %s', start_step)
+
+  for s in range(start_step, loop_steps):
```

Comment:
Maybe `range(start_step + 1, loop_steps + 1)`?
Then we would e.g. not checkpoint at the very first step but after `checkpoint_frequency` steps for the first time.